src/scalarstats.jl

```diff
@@ -390,6 +390,47 @@ function entropy{T<:Real}(p::AbstractArray{T}, b::Real)
     return entropy(p) / log(b)
 end
 
+"""
+    renyientropy(p, α)
+
+Compute the Rényi (generalised) entropy of order `α` of an array `p`.
```

Comment:
> Compute the generalized Rényi entropy of order `α` of a real-valued array `p`.

Unless that isn't what you meant by "(generalized)". Also we should probably stay consistent with the American English spelling used in other places in the package, i.e. use "z" instead of "s" in words like "generalized."